tagilmo/VereyaPython/timestamped_video_frame.py

```diff
@@ -10,8 +10,8 @@
 import numpy.typing as npt
 import cv2
 import io
-import PIL.Image as Image
 import os
+import platform
```

Comment:
Do we still need `import platform` here if there is no platform-dependent code after all?

Reply:
@Necr0x0Der  I forgot it. Sorry. Now it's gone